create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -47,14 +47,22 @@ You can use the cluster resolver to specify a remote pipeline or task that is de
 include::modules/op-resolver-cluster-config.adoc[leveloffset=+2]
 include::modules/op-resolver-cluster.adoc[leveloffset=+2]
 
-[id="resolver-git_{context}"]
-== Specifying a remote pipeline or task from a Git repository
+[id="resolver-git-anon_{context}"]
+== Specifying a remote pipeline or task with anonymous Git cloning
 
-You can use the Git resolver to specify a remote pipeline or task from a Git repostory. The repository must contain a YAML file that defines the pipeline or task. The Git resolver can access a repository either by cloning it anonymously or else by using the authenticated SCM API.
+You can use the Git resolver to access a remote pipeline or task from a Git repository. The repository must include a YAML file that defines the pipeline or task. For anonymous access, you can clone repositories with the resolver without needing authentication credentials.
 
 include::modules/op-resolver-git-config-anon.adoc[leveloffset=+2]
+include::modules/op-resolver-git-anon.adoc[leveloffset=+2]
+
+[id="resolver-git-scm_{context}"]
+== Specifying a remote pipeline or task with authenticated SCM API
```

Comment:
I wonder about the usage of "SCM API".. SCM stands for "source code management" I think, and this use (upstream I guess) comes from the fact that we use https://github.com/jenkins-x/go-scm as a library to do those authenticated clone.

We could just say with authenticated API or something like that. In a gist, the "authenticated fetch on the git repository is done through the respective API (github, gitlab, …)".

Reply:
So different API depending on the git provider (vs using the "standard" git clone/fetch mechanisms)